documentation/c01-whatis.sil

```diff
@@ -82,11 +82,105 @@ SILE does not benefit from the large ecosystem and community that has grown up a
 
+\section{SILE versus Typst and other new alternatives}
+
+Almost exactly one decade after SILE development started, a new competitor to LaTeX entered the scene with a bang:
+\footnote{Days of front-page coverage on Hacker News, Lobsters, and the like propelled it past 10k GitHub stars in the first 6 days of being public, a milestone SILE is still very far from reaching.}
+Typst.
+There is no shame in stating that Typst is a great piece of software and a worthy competitor to SILE as well as well as LaTeX.
+Typst is (very) fast, well documented, and relatively versatile.
+While both typesetting engines cover much of the same ground, the respective focus and implementation choices of each are so varied that direct comparisons don’t always have clear winners.
+At the end of the day the specific use case and workflow preferences are at least as relevant as the overall merits of each engine.
+
+When Typst was first conceived, SILE had been around the block in production for a while; but had only really been used for prose.
+Its math support was barely at a proof-of-concept stage and definitely not robust enough for scientific publishing work.
+This niche was exactly the felt need of Typst’s developers; the major focus right out of the gate was math and science papers.
+In the first releases it was more than a little bit rough in the prose department.
+Since then SILE’s math chops have greatly expanded to cover the majority of MathML.
+Typst’s feature set for handling prose have also improved.
+That being said the initial focus and energy of each project is still very visible.
+
+\begin{itemize}
+
+\item{\strong{Field of focus:}
+	SILE was initially conceived to handle hard problems in automated typesetting of books in many world orthographies.
+	Typst has some multi-lingual support but does not have robust support for the wide variety of typesetting issues posed by the world’s scripts and languages to the extent SILE does.
+	Typst was initially conceived for work in the academic sciences, and the robustness of the math and figure APIs directly reflects this.
+	SILE has math support but it has not been exercised to nearly the extent of Typst’s.}
+
+\item{\strong{Input formats:}
+	SILE was conceived to handle multiple input formats; out of the box it supports a LaTeX-like syntax but also includes an XML reader suitable for inputting other formats.
+	In addition writing input drivers is relatively easy and 3rd party ones for Markdown, DJot, and others have been developed.
+	Typst on the other hand developed their own input format, loosely based on Markdown but distinct and unique; it does not support input in anything else.
+	Pandoc support allows some other formats to be converted and imported, but not worked with natively.}
+
+\item{\strong{Scripting:}
+	SILE exposes (and indeed is actually mostly implemented in) Lua, an industry standard embedded scripting language with an already large ecosystem of code and packages.
```

Comment:
"an already large ecosystem": Compared to what? If SILE had stuck to JS, or been implemented in Python (for instance), there would be trillions of existing components to use for plenty of fancy things. The Lua ecosystem of reusable components is... very very very small, actually, no?